timemachine/cpp/src/kernels/k_fixed_point.cuh

```diff
@@ -17,7 +17,7 @@ RealType __device__ __forceinline__ FIXED_TO_FLOAT_DU_DP(unsigned long long v) {
 // (ytz): courtesy of @scottlegrand/NVIDIA, even faster conversion
 // This was original a hack to improve perf on Maxwell, that is now needed for Ampere
 long long __device__ __forceinline__ real_to_int64(float x) {
-#if __CUDA_ARCH__ == 860 || __CUDA_ARCH__ == 750
+#if __CUDA_ARCH__ >= 750
```

Comment:
Okay I'm confused, I double checked https://en.wikipedia.org/wiki/CUDA

```
T4 is 7.5
RTX 2080 is 7.5
A4000 is 8.6
RTX 3090 is 8.6
A10G is 8.6
```

Are you saying that A10G performs better with this *disabled*?

Reply:
No, it performs better with it enabled. I just wanted to move away from two versions explicitly stated. But need to benchmark an A100 to be sure that this works, for some reason I thought we had benchmarked an 8.0 card already

Reply:
IIRC A100 performs *better* with this *disabled* since it has full double precision ALUs.

Reply:
Been having a hard time getting A100s to test this on